feat: UIG-2411 - vl-input-slider verbeteringen

ksvlaanderen opened the pull request:
vl-input-slider aria labels toegevoegd
vl-input-slider min/max validatie toegevoegd

Goldflow commented on apps/storybook-e2e/src/e2e/components/input-slider/vl-input-slider.stories.cy.ts:
```diff
@@ -1,19 +1,29 @@
+    it('should be accessible', () => {
+        cy.visit(inputSliderUrl);
+
+        cy.injectAxe();
```
goeie toevoeging, kunnen we gebruiken overal
eventueel custom command maken die visit & injectAxe combineert
beslis zelf maar!

Goldflow commented on libs/components/src/lib/input-slider/stories/vl-input-slider.stories-arg.ts:
```diff
@@ -37,6 +28,16 @@ export const inputSliderArgTypes: ArgTypes<typeof inputSliderArgs> = {
             defaultValue: { summary: 0 },
         },
     },
+    value: {
```
goeie wijziging!

Goldflow reviewed (approved):
ziet er goed uit
goede toevoegingen ivm die accessibility & verandering initialValue > value

zoals besproken, opmerking ivm eventueel custom `visitWithAccessibility`(of een anderee naam) naar eigen inzicht te verwerken

ksvlaanderen committed:
feat: UIG-2411 - vl-input-slider verbeteringen

vl-input-slider aria labels toegevoegd
vl-input-slider min/max validatie toegevoegd

kspeltix commented:
:tada: This PR is included in version 1.5.0 :tada:

The release is available on [GitHub release](https://github.com/milieuinfo/uigov-web-components/releases/tag/v1.5.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
